ssl/ssl_rsa.c

```diff
@@ -904,7 +920,7 @@ int SSL_CTX_use_serverinfo_file(SSL_CTX *ctx, const char *file)
     int ret = 0;
     BIO *bin = NULL;
     size_t num_extensions = 0, contextoff = 0;
-    unsigned int version = 0;
+    unsigned int version;
```

Comment:
At this point, `version` could move entirely inside the `for` loop below, since it isn't used anywhere else.